src/SprykerEco/Zed/Econda/EcondaConfig.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\Econda;
+
+use Exception;
+use Spryker\Shared\Application\ApplicationConstants;
+use Spryker\Zed\Kernel\AbstractBundleConfig;
+use SprykerEco\Shared\Econda\EcondaConstants;
+
+class EcondaConfig extends AbstractBundleConfig
+{
+    const ECONDA_CSV_DELIMITER = '|';
+    const ECONDA_CSV_CATEGORY_DELIMITER = '^^';
+
+    /**
+     * @return string
+     */
+    public function getHostYves()
+    {
+        return $this->get(ApplicationConstants::HOST_YVES);
+    }
+
+    /**
+     * @return string
+     */
+    public function getFileExportPath()
+    {
+        return $this->get(EcondaConstants::ECONDA_CSV_FOLDER_PATH);
+    }
+
+    /**
+     * @param string $pdoEcondaQueryName
+     * @param string $dbEngineName
+     *
+     * @throws \Exception
+     *
+     * @return string
+     */
+    public function getPdoEcondaQueryClassName($pdoEcondaQueryName, $dbEngineName)
+    {
+        $data = [
+            'MySql' => [
+
+            ],
+            'PostgreSql' => [
+
+            ],
+        ];
+
+        if (!isset($data[$dbEngineName][$pdoEcondaQueryName])) {
+            throw new Exception('Invalid PdoEcondaQueryName name: ' . $pdoEcondaQueryName);
+        }
```

Comment:
I didn't quite understand this part… . In current implementation, and `Exception` will be thrown **always**, right?

Reply:
@alex-galych what kind of workaround do you suggest?

Reply:
@gerner-spryker please advice

Reply:
This is a business logic implementation error. I can't help you with this, you need to evaluate the business requirements and figure out what the code was supposed to do; the current implementation definitely feels wrong.

Reply:
Ok, so it's possible to have exception in the Config.php if you expect that it will be setup during the integration, is this the case?